src/pages/farewell.astro

```diff
@@ -0,0 +1,124 @@
+---
+import { BaseLayout } from '~layouts';
+
+const firstHeadingId = 'farewell';
+---
+
+<BaseLayout title="A Memorable Journey's End" skipTargetId={firstHeadingId}>
+	<div class="l-center">
+		<h1 id="{firstHeadingId}" class="h2">A Memorable Journey's End</h1>
+		<p>Dear Collab Lab Community and friends of The Collab Lab,</p>
+		<p>
+			After much thought, we’ve decided to close The Collab Lab at the end of
+			this year. It was an incredibly tough decision, but we believe it’s the
+			right move. Before we wrap up, we have one final cohort ahead of us, and
+			we hope you'll help make it unforgettable! ⭐️
+		</p>
+		<p>
+			Over the years, we've achieved amazing things together.
+			<a href="#developers" style="color: var(--color-blue-mid)"
```

Comment:
Do we need to set the color of the link explicitly here?

And, this should probs link to `/developers/` not `#developers`, right?